conbench/templates/compare-entity.html

```diff
@@ -163,34 +163,72 @@
             </li>
           {% endfor %}
           <li class="list-group-item active">Tags</li>
-          {% for k,v in baseline.tags.items() %}
-            <li class="list-group-item" style="overflow-y: auto;">
-              <b>{{ k }}</b>
-              {% if v is not none %}
-                <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
+          {% for field in tags_fields %}
+            {% if field != "id" and (field not in baseline.tags or field not in contender.tags or baseline.tags[field] != contender.tags[field]) %}
+              <li class="list-group-item list-group-item-danger" style="overflow-y: auto;">
+            {% else %}
+               <li class="list-group-item" style="overflow-y: auto;">
+            {% endif %}
+              {% if field in baseline.tags %}
+                <b>{{ field }}</b>
+                {% if baseline.tags[field] is not none %}
+                  <div align="right" style="display:inline-block; float: right;">{{ baseline.tags[field] }}</div>
+                {% endif %}
+              {% else %}
+                <div class="invisible">*</div>
```

Comment:
Using `<div class="invisible">*</div>` because I could not figure out how to make an empty cell which also has correct hight. 

Let me know if you have better ideas.